### PR TITLE
chore: general improvements in dapp example

### PR DESCRIPTION
**Pull request by ignaciosantise**
## Changelog

- Added new app icons
- Added missing sizes for some icons
- Added Colors file with theme colors
- Code improvements
- Unified criteria for component declarations
- Changed app name
- Added some missing wallets to json file

### Android app icon
Screen Shot 2023-03-22 at 14 53 44

### iOS app icon
Screen Shot 2023-03-22 at 14 53 41

### CODE REVIEW

File: dapps/v2Explorer/src/components/NavigationHeader.tsx [hunk 1]
```diff
@@ -76,11 +83,17 @@ const styles = StyleSheet.create({
...
-    color: '#1f1f1f',
+    color: LightTheme.foreground1,
     fontSize: 20,
     lineHeight: 24,
   },
   titleDark: {
-    color: 'white',
+    color: DarkTheme.foreground1,
+  },
+  // Handle better when themes are added
```
Comment (crypblizz8) [inline dapps/v2Explorer/src/components/NavigationHeader.tsx:93]: Small: Rm comment

File: dapps/v2Explorer/src/components/QRCode.tsx [hunk 2]
```diff
@@ -2,15 +2,15 @@ import React, {useMemo} from 'react';
 import {Svg} from 'react-native-svg';
 import {View, Image, StyleSheet} from 'react-native';
 import {QrCodeUtil} from '../utils/QrCode';
-import WCLogo from '../assets/WCGradientLogo.png';
+import WCIsotype from '../assets/WCIsotype.png';
 
 interface Props {
   uri: string;
   size: number;
   theme?: 'light' | 'dark';
```
Comment (crypblizz8) [inline dapps/v2Explorer/src/components/QRCode.tsx:10]: Do these strings need to be tied to a const?

File: dapps/v2Explorer/src/components/QRCode.tsx [hunk 3]
```diff
@@ -2,15 +2,15 @@ import React, {useMemo} from 'react';
...
 }
 
-function QrCode({uri, size, theme = 'light'}: Props) {
+function QRCode({uri, size, theme = 'light'}: Props) {
```
Comment (crypblizz8) [inline dapps/v2Explorer/src/components/QRCode.tsx:13]: Same as this comment. I.e. we have a const with lightTheme = 'light' in a utils folder.

File: dapps/v2Explorer/src/constants/Colors.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
+// Will be better used when Themeing is added
```
Comment (crypblizz8) [inline dapps/v2Explorer/src/constants/Colors.ts:1]: Still keep comment?

File: dapps/v2Explorer/src/utils/QrCode.tsx [hunk 5]
```diff
@@ -44,8 +45,10 @@ export const QrCodeUtil = {
     logoSize: number,
     theme: 'dark' | 'light',
```
Comment (crypblizz8) [inline dapps/v2Explorer/src/utils/QrCode.tsx:46]: same comment as const as before.

File: dapps/v2Explorer/src/constants/Colors.ts (hunk 4 above)
Comment (ignaciosantise) [reply dapps/v2Explorer/src/constants/Colors.ts:1]: i can remove it, but my idea is to handle the theme using a context at top of the modal

File: dapps/v2Explorer/src/components/QRCode.tsx (hunk 2 above)
Comment (ignaciosantise) [reply dapps/v2Explorer/src/components/QRCode.tsx:10]: i could create a type with those constants, following RN core code

Screen Shot 2023-03-27 at 12 40 32

**Pull request merged**